### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by m05e5**
# To Do list: list structure

- Set up a new project with webpack that is based on the webpack exercise you have already completed.
- Create an index.html file and write your HTML markup here
- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
    * description [string].
    * completed [bool].
    *  index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <title>Home</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+  </head>
+  <body>
+    <div class="todo-list">
+      <div class="head-section">
+        <p class="head-title">Today's To Do</p>
+        <img src="../assets/reset.png" alt="" class="head-icon">
```
Comment (mricanho) [inline src/index.html]: - [ ] Double-check the style of this icon, this happens when I run your app:

image

Target style:

image

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    const li = document.createElement('li');
+    li.classList.add('todo');
+    // create checkbox
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = todo.completed;
+    li.appendChild(checkbox);
+    // create description
+    const desc = document.createElement('p');
+    desc.innerText = todo.description;
```
Comment (mricanho) [inline src/index.js:54]: - [ ] After the todo.description, you need to add a three dots icon, this is in the target design

Your design lacks the three dots icon:

image

Target design:

image

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @m05e5, 

Good job so far! 🔥🔥 Your JS code it's almost done.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] Remember that you need to add a complete README file, especially the setup and installation sections are super important to run correctly your project, please add it.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

almost there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "todo_list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
+    "build": "webpack"
```
Comment (mricanho) [inline package.json:9]: - [ ] You need to add a production and development configuration, even you have a warning when you run "npm start:

webpack

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._
- [ ] As requested in the previous review kindly add the `three-dot-icon`.

Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (mikenath223) [reply src/index.js:54]: > * [ ]  After the todo.description, you need to add a three dots icon, this is in the target design
> 
> Your design lacks the three dots icon:
> 
> image
> 
> Target design:
> 
> image

- [ ] As requested in the last review, please add the `three-dots-icon` as seen in the design.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    li.classList.add('todo');
+    const liDiv = document.createElement('div');
+    liDiv.classList.add('li-div');
+    // create checkbox
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = todo.completed;
+    liDiv.appendChild(checkbox);
+    // create description
+    const desc = document.createElement('p');
```
Comment (mikenath223) [inline src/index.js:53]: > * [ ]  After the todo.description, you need to add a three dots icon, this is in the target design
> 
> Your design lacks the three dots icon:
> 
> image
> 
> Target design:
> 
> image

- [ ] As requested in the last review, please add the `three-dots-icon` as seen in the design.

Comment (m05e5) [reply src/index.js:53]: i added the 3 dots please check at my last commit

Comment (m05e5) [reply src/index.js:53]: image

Comment (m05e5) [reply src/index.js:53]: image

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**